anathema-backend/src/tui/style.rs

```diff
@@ -84,11 +89,19 @@ impl CellAttributes for Style {
         };
 
         match colour {
-            Some(Color::Rgb { r, g, b }) => Some(Hex::from((r, g, b))),
+            Some(Color::Rgb(r, g, b)) => Some(Hex::from((r, g, b))),
             None | Some(_) => None,
         }
     }
 
+    fn get_color(&self, key: &str) -> Option<Color> {
+        match key {
+            "foreground" => self.fg,
+            "background" => self.fg,
```

Comment:
i believe that `background => self.fg` is incorrect

```suggestion
            "background" => self.bg,
```